packages/pharos-site/src/components/statics/color/ColorCombos.tsx

```diff
@@ -1,6 +1,8 @@
 import type { FC, CSSProperties } from 'react';
 import { colorBox } from './ColorCombos.module.css';
-import tokens from '@ithaka/pharos/lib/styles/tokens';
+// import tokens from '@ithaka/pharos/lib/styles/tokens.js';
+import tokens from '../../../../../pharos/lib/styles/tokens';
```

Comment:
So it looks like we are importing the tokens from  the `pharos` source code directly now instead of using the already installed `@ithaka/pharos` package. Is that desired? It creates tighter coupling between `@ithaka/pharos` and `@ithaka/pharos-site`, but it allows us to bridge breaking changes like this in one PR.

If this is desired It's almost like we should uninstall `@ithaka/pharos` from `@ithaka/pharos-site` completely.

If it's not desired, then I wonder if we should leave `@ithaka/pharos-site` alone for now, and circle back to upgrade `@ithaka/pharos` to v13 once it's available. To me this feels like the desired approach, but wanted to raise the point.

Reply:
@eslawski Without this change to the import, the feature branch kept failing to build. However seeing as we were previously using `@ithaka/pharos`, I was a little hesitant to move forward with this change without a second set of eyes. I think this should work but wanted your opinion as well 👍🏻

Reply:
So my inclination is that we should not be modifying `@ithaka/pharos-site` at this time. This is because `@ithaka/pharos-site` installs `@ithaka/pharos`. So if we focus on only making the updates to `@ithaka/pharos`, we can come back to `@ithaka/pharos-site` later once v13 is released.

Reply:
I might request a sanity check this approach though from other @ithaka/pharos-maintainers

Reply:
Aha! Makes sense. So revert the `isOnBackground` changes in `/pharos-site/` back to `onBackground` and let it get updated from `@ithaka/pharos/` instead? @eslawski

Reply:
Ya, more specifically I am suggesting we revert any and all changes made in `/pharos-site/`. That I understand will be something we come back and do once the next major version of pharos is cut.